server/portal/apps/workspace/api/views.py

```diff
@@ -247,21 +247,31 @@ def post(self, request, *args, **kwargs):
 
             if settings.DEBUG:
                 wh_base_url = settings.WH_BASE_URL + '/webhooks/'
-                # jobs_wh_url = settings.WH_BASE_URL + reverse('webhooks:jobs_wh_handler')
+                jobs_wh_url = settings.WH_BASE_URL + reverse('webhooks:jobs_wh_handler')
             else:
                 wh_base_url = request.build_absolute_uri('/webhooks/')
-                # jobs_wh_url = request.build_absolute_uri(reverse('webhooks:jobs_wh_handler'))
+                jobs_wh_url = request.build_absolute_uri(reverse('webhooks:jobs_wh_handler'))
 
             job_post['parameterSet']['envVariables'] = job_post['parameterSet'].get('envVariables', []) + [{'key': '_webhook_base_url', 'value':  wh_base_url}]
 
             portal_name = settings.PORTAL_NAMESPACE
             job_post['tags'] = job_post.get('tags', []) + [portal_name]
 
-            # TODOv3 Webhooks/notifications continues
-            # job_post['notifications'] = [
-            #     {'url': jobs_wh_url,
-            #      'event': e}
-            #     for e in settings.PORTAL_JOB_NOTIFICATION_STATES]
+            # ttlMinutes of 0 corresponds to max default (1 month)
+            job_post["subscriptions"] = [
+               {
+                    "description": "Portal job status notification",
+                    "enabled": True,
+                    "eventCategoryFilter": "JOB_NEW_STATUS",
+                    "ttlMinutes": 0,
```

Comment:
```suggestion
            # ttlMinutes of null corresponds to max default (1 month)
            job_post["subscriptions"] = [
               {
                    "description": "Portal job status notification",
                    "enabled": True,
                    "eventCategoryFilter": "JOB_NEW_STATUS",
```

Reply:
@rstijerina new update from tapis team allows 0 for ttlMinutes. I tested and it looks to be working as intended. Can you try testing it again?

Reply:
do we need to set to explicitly set to 4 weeks?   I understood this comment https://tacc-team.slack.com/archives/C0307MGPCP2/p1673478074341959 to mean the default is 1 week.

Reply:
@nathanfranklin Good catch, that sounds right. @shayanaijaz Do we want 4 weeks specifically, or are we ok with 1 week? I don't think jobs can run that long, so that may be ok.

Reply:
Right, if we are ok with 1 week then I can remove the ttlMinutes field entirely which should set the max time to 1 week

Reply:
I think 1 week is acceptable.